index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Html forms</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="./assets/css/style.css">
+    </head>
+    <body>
+        <header class="flex-container flex-direction-row-reverse">
+            <button class="btn pr10">Sign In</button>
+            <div class="pr10">
+                <p>Already have an Intuit account?</p>
+                <a>I forgot my User ID or Password</a>
```

Comment:
You can add the `href="#"` attribute to this.

Reply:
added thank you